### PR TITLE
Update personal url.

### PR DESCRIPTION
**Pull request by andrewhughes3000**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,8 +43,8 @@
         // editors, add as many as you like
         // only "name" is required
         editors: [
-          { name: "Andrew Hughes", url: "FIXME_PERSONAL_URL",
-            company: "", companyURL: "FIXME_COMPANY_URL" },
+          { name: "Andrew Hughes", url: "https://idimmusings.com/",
```
Comment (msporny) [inline index.html:46]: This looks good, thanks for filling in your personal website name.

File: index.html [hunk 2]
```diff
@@ -43,8 +43,8 @@
...
+            company: "ITIM Consulting", companyURL: "https://idimmusings.com/" },
```
Comment (msporny) [inline index.html:47]: Hmm, don't know if you want to use your personal website as the company one or vice versa... are you sure?

**Review by msporny — APPROVED** [review]
LGTM. Minor nits on URL.

File: index.html (hunk 2 above)
Comment (andrewhughes3000) [reply index.html:47]: Yes, I'm sure.

**Pull request merged**